src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -533,7 +533,10 @@ impl ThinPool {
         );
 
         let original_state = self.cached(|pool| ThinPoolState {
-            usage: pool.total_physical_used().map(|s| s.bytes()).ok(),
+            usage: pool
```

Comment:
Looks right to me. Could you abstract this expression into a method in the thinpool and make ```StratPool::total_physical_used()``` call that method? I'm worried about the two code locations getting out of sync again, otherwise.

Reply:
That sounds reasonable to me. I was considering that when I saw the note above the implementation in `pool.rs`.

Reply:
Resolved!